sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -681,6 +679,8 @@ export interface RetryOptions {
     timeoutInMs?: number;
 }
 
+export { RheaMessage }
```

Comment:
Why export this at all?

Reply:
Fine, service-bus can import it directly!